_redirects

```diff
@@ -193,7 +193,7 @@
 /docs/upgrading-to-014	/docs/guides/migration-guide/upgrading-to-014	302
 /docs/use-an-existing-project	/docs/building-a-dbt-project/dbt-projects/use-an-existing-project	302
 /docs/using-custom-aliases	/docs/building-a-dbt-project/building-models/using-custom-aliases	302
-/docs/using-custom-database	/docs/building-a-dbt-project/building-models/using-custom-database	302
+/docs/using-custom-database	/docs/building-a-dbt-project/building-models/using-custom-databases	302
```

Comment:
Has this been a broken link the whole time?

Reply:
All of our links broken when we moved from readme to docusaurus, which is why we have such a long `_redirects` file!